clr-us/src/components/course-page/view-problem.tsx

```diff
@@ -15,7 +16,9 @@ export const ViewProblem: React.FC = () => {
   const problemType = useGetProblemType()
   const problemId = useGetProblemId()
   const [problem, setProblem] = useState<Problem | undefined>(undefined)
-  const [inReview, setInReview] = useState(false)
+  const [status, setStatus] = useState<ProblemStatus>(ProblemStatus.Posted)
+
+  const forceRefresh = () => problemService.getProblem(problemId, setProblem)
```

Comment:
@judy-n I was hoping to get your eyes on this. Passing a refresh function down to children which can force parent component to re-fetch problem after it's edited. Is this ok? Or an Anti-Pattern?

Reply:
Judy reviewed on my computer 👍